### PR TITLE
Solving PR #7 - plots

### LINKED ISSUE #7: Plot the loss and accuracy given a neural net for our problem
Once a pipeline to save train and test results are done, we need to figure out a way to plot the results of a training/test phases.

### PR DESCRIPTION
**Pull request by dsbuddy**
Once a pipeline to save train and test results are done, we need to figure out a way to plot the results of a training/test phases.

### CODE REVIEW

**Comment by isamu-isozaki** [comment]
I was thinking about including that in the pipeline

**Comment by ya332** [comment]
The code looks good, but a few previous PRs got approved so you need to update the current folder structure and add the plots to the binary_search_networks/pipeline.py instead of the train.py @dsbuddy

**Review by ya332 — CHANGES_REQUESTED** [review]
I think adding the plots to the binary_search_networks/pipeline.py instead of the train.py would make the code more modular. WDYT?

**Review by ya332 — APPROVED** [review]

**Comment by ya332** [comment]
LGTM!
